doc/manifest/schema/1.0.0/defaultLocale.md

```diff
@@ -0,0 +1,208 @@
+ </details>
+ 
+#### PackageUrl
+<details>
+  <summary>The package home page</summary>
+
+  #### Optional Field
+  
+  This key represents the web site for the package.
+ </details>
+
+ #### License
+<details>
+  <summary>The package license</summary>
+
+  #### Required Field
+
+  This key represents the license governing the use and or distribution for the product. This could be an open source license, or a commercial license.
+ </details>
+
+ #### LicenseUrl
+<details>
+  <summary>The license page</summary>
+
+  #### Optional Field
+  
+  This key represents the license web site or specific web page provided the publisher or ISV. If there is a license web site or specific web page for the package it is preferred over a generic license page for the publisher.
+  
+  If this is a link to the license file for an open source project, it should be specific to the version for the package. Some open source projects change their license over time.
+ </details>
+
+ #### Copyright
+<details>
+  <summary>The package copyright</summary>
+
+  #### Optional Field
+
+  This key represents the copyright for the package.
+ </details>
+
+ #### CopyrightUrl
+<details>
+  <summary>The package copyright page</summary>
+
+  #### Optional Field
+  
+  This key represents the copyright web site or specific web page provided the publisher or ISV. If there is a copyright web site or specific web page for the package it is preferred over a generic copyright page for the publisher.
+  
+  If this is a link to the copyright file for an open source project, it should be specific to the version for the package. Some open source projects change their copyright over time.
+ </details>
+
+ #### ShortDescription
+<details>
+  <summary>The short package description</summary>
+
+  #### Required Field
+
+  This key represents the description for a package. It is intended for use in `winget show` to help a user understand what the package is.
+
+  >Note: This should be something descriptive about what the package does, and it should not simply state something like "&lt;package name&gt; installer" or "&lt;package name&gt; setup".
+ </details>
+ 
+ #### Description
+<details>
+  <summary>The full package description</summary>
+
+  #### Optional Field
+
+  This key represents the full or long description for a package. It is *not* currently used in the Windows Package Manager.
+
+  >Note: This was included for future integration with the Microsoft Store source to provide the ability to display the full package description.
+ </details>
+ 
+ #### Moniker
+<details>
+  <summary>The most common package term</summary>
+
+  #### Optional Field
+
+  This key represents the most common term users would search for when installing or upgrading a package. If only one package uses this moniker, then the install, list and upgrade command will match with this package.
```

Comment:
nit: IIRC the moniker is only 3rd in priority when we do package matching. This statement seems like over simplifying our matching logic.